### DISCUSSION TITLE
SONOFF SNZB-04 no link on the Zigbee map

### DISCUSSION THREAD

**Discussion by hugalafutro** [Koenkk/zigbee2mqtt#9442] [Devices]
Hi,
some time after I first added SONOFF SNZB-04 door sensor to my zigbee network, it stopped showing the line connecting it to a nearby router / coordinator, it just hovers there alone. Other end-devices I have show their lines. Everything seems to be working fine, I just wonder whether it's not indicative of any other issue.

like this:
![Screenshot 2021-11-01 152355](https://user-images.githubusercontent.com/30209689/139696439-cc40255b-8882-43c6-8ca3-5356b69331ad.png)

Thanks!

**Comment by francisp2** [comment]
Seems to be a 'feature' of the Sonoff devices. My SNZB-03's don't show a link either, however, they never fail to work.

**Comment by dsduarte** [comment]
i have posted a similar doubt before and somebody answer that could be that the device got in "dorment" state... I thought it was reasoneable as its a passive deveice... i mean, the device doesn't need to be "online" as it only need to report open/close state.
